README.md

```diff
@@ -1,2 +1,106 @@
+[![CircleCI](https://circleci.com/gh/EasyEdu/enrollment/tree/master.svg?style=svg)](https://circleci.com/gh/EasyEdu/enrollment/tree/master)
+
 # enrollment
-A JS client, to allow enroll students on EasyEdu courses by integrating it on your sites.
+A JS client to allow enroll students on EasyEdu course's classes by integrating it on your sites.  
+  
+You can integrate this client on your course landing page for example, this way you can collect lead information, and enroll then on your EasyEdu course's class.
+
+## Usage  
+
+First of all, add this tag to your html page, inside the `<head>` element:
+```html
+  <script type="text/javascript" src="https://scripts.easyedu.co/enrollment.js"></script>
+```
+
+Then, in a script you can call:
+
+```js
+<script type="text/javascript">
+  window.easyedu.enroll('your-class-auth-token-here', {
+    email: 'new-student-email@mail.com',
+    full_name: 'Full Name'
+  }, sucessCallback, errorCallback);
+</script>
+  ```
+  
+Usually you will call this code inside a sign up form callback.
+
+### Arguments
+The first argument for the window.easyedu.enroll function is an authorization token of the class you are trying to enroll your users to. You can contact us to have a authorization token for your classes at `contato@ieducativa.com.br`  
+
+The second argument for the window.easyedu.enroll function is JS object containing user properties:  
+* email: the email of the user that will be enrolled to the class (required);  
+* full_name: the full name of the user that will be enrolled to the class (optional).
+
+The third argument for the window.easyedu.enroll function is a function that will be called when the user is sucessfully enrolled to the class. The callback will receive the request response as argument.  
+
+The fourth argument for the window.easyedu.enroll function is a function that will be called when the enrollment request to our API gets some error as response. The callback will receive the request response as argument.  
+
+### Client errors
+ * If you don't pass the authorization key, you will get the error: `Uncaught authKey param is missing.`  
+ * If you pass a authorization token that is not a string, you will get the error: `Uncaught authKey must be a string.`  
+ * If you don't pass the user properties object, you will get the error: `Uncaught params param is missing.`  
+ * If you pass a user properties object that is not an object, you will get the error: `Uncaught params param must be a object.`
+
+
+### API errors
+If the given authorization key is not valid, you will get the following error as on the errorCallback:
```

Comment:
Please include the following words here and in the next similar sections:

If the given authorization key is not valid, you will get the following error as **the first argument** on the errorCallback: